packages/grafana-toolkit/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "../tsconfig.json",
-  "include": ["src/**/*.ts"],
+  "include": ["src/**/*.ts", "../../public/app/types/jquery/*.ts"],
```

Comment:
should we just move the jquery types to toolkit?

Reply:
Nah, I don't think so. We would have to move those to grafana/data as well as it uses them too. I feel referencing those from core is not that bad here especially those are needed only on compilation time which happens in core env anyways.